lib/rollbar/request_data_extractor.rb

```diff
@@ -57,6 +57,26 @@ def extract_request_data_from_rack(env)
       data
     end
 
+    def extract_custom_data_from_rack(env)
+      controller = env["action_controller.instance"]
+      if !controller
+        Rollbar.configuration.custom_data_method.call
+        return
+      end
+
+      # Only extract custom data if we've defined a custom_data_method, or specific values on the controller
+      controller_values = Rollbar.configuration.custom_values[controller.class.name.parameterize] || []
+      custom_data_block = Rollbar.configuration.custom_data_method
+      return {} if controller_values.empty? && !custom_data_block
+
+      custom_data = custom_data_block ? controller.instance_exec(&custom_data_block) : {}
```

Comment:
We can move this logic to some controller method so the controllers are responsible to return the correct logging data. This can be then simpler.